Modules/Core/Common/include/itkMersenneTwisterRandomVariateGenerator.h

```diff
@@ -345,10 +346,10 @@ MersenneTwisterRandomVariateGenerator::Initialize(const IntegerType seed)
   IntegerType * s = state;
   IntegerType * r = state;
 
-  *s++ = seed & 0xffffffffUL;
+  *s++ = seed;
   for (IntegerType i = 1; i < MersenneTwisterRandomVariateGenerator::StateVectorLength; ++i)
   {
-    *s++ = (1812433253UL * (*r ^ (*r >> 30)) + i) & 0xffffffffUL;
+    *s++ = uint32_t{ (uint32_t{ 1812433253 } * (*r ^ (*r >> 30)) + i) };
```

Comment:
Note that the effect of the original `UL` suffix is platform dependent. On Linux, it's typically 64 bits, and on Windows 32 bits. _Probably_ it doesn't matter, but it's much clearer to explicitly convert to `uint32`.